src/app/organizations/organization-details/manage-projects/manage-projects.component.html

```diff
@@ -9,22 +7,39 @@ <h2 class="app-page-subtitle">Manage projects</h2>
 
     <ul class="list-group">
         <li *ngFor="let project of projects"
-            class="list-group-item explainer-mini">
-
-                <span class="float-left hover-bg cursor-pointer"
-                      routerLink="manage_project/{{ project.uuid }}">
-                    {{ project.name }}
-                </span>
-
-            <span *ngIf="project.active" (click)="toggleProject(project.uuid, projects)"
-                  class="float-right hover-fg cursor-pointer">
-                    <i class="fas fa-eye"></i>
-                </span>
-
-            <span *ngIf="!project.active" (click)="toggleProject(project.uuid, projects)"
-                  class="float-right hover-fg cursor-pointer">
-                    <i class="fas fa-eye-slash"></i>
-                </span>
+            class="list__item">
+            <div class="item__top">
+                <div class="top__left">
+                    <img src="{{ project.main_image}}" alt="Project image">
+                </div>
+                <div class="top__right">
+                    <div class="item__title"
+                         routerLink="manage_project/{{ project.uuid }}">
+                        {{ project.name }}
+                    </div>
+                    <div class="item__funding">
+                        <b>Funding </b>{{ project.start_date | date: "longDate" }} - {{ project.end_date | date: "longDate" }}
+                    </div>
+                </div>
+            </div>
+            <div class="item__bottom">
+                <div class="item__progress-bar">
+                    <!-- TODO: Implement progress bar here when data is available -->
+                    Progress bar
+                </div>
+                <div class="item__toggle">
+                    <span *ngIf="project.active" (click)="toggleProject(project.uuid, projects)"
```

Comment:
Let me check if I understood you correctly. You would like to add spinners on every instance of ngFor loop (li element) instead of overlaying the entire container with spinner?

Reply:
Yeah. So when I click the publish/unpublish button it spins itself... UX wise it's much better